src/instance.ts

```diff
@@ -35,6 +35,27 @@ import {
 } from './table';
 import {CallOptions} from 'google-gax';
 
+import {Bigtable} from '.';
+import {google} from '../proto/bigtable';
+import {ServiceError} from '@grpc/grpc-js';
+
+export interface CreateInstanceOptions extends CallOptions {
```

Comment:
`CreateInstanceOptions` should not extend from `CallOptions` rather it should have a property `gaxOptions: CallOptions`. (Can extend `OptionsIntrface` from earlier table PR)